src/hooks/restrict-to-owner.js

```diff
@@ -0,0 +1,42 @@
+import errors from 'feathers-errors';
+
+const defaults = {
+  idField: '_id',
+  ownerField: 'userId'
+};
+
+export default function(options = {}){
+  return function(hook) {
+    if (hook.type !== 'before') {
```

Comment:
Maybe then we even want to make this the same as the `restrict-to-self` hook from the guide.